src/ScriptTools.sol

```diff
@@ -74,7 +77,39 @@ library ScriptTools {
      * @return config The raw json text of the config.
      */
     function loadConfig() internal returns (string memory config) {
-        config = vm.envOr("FOUNDRY_SCRIPT_CONFIG_TEXT", readInput(vm.envString("FOUNDRY_SCRIPT_CONFIG")));
+        config = vm.envOr("FOUNDRY_SCRIPT_CONFIG_TEXT", string(""));
+        if (eq(config, "")) {
+            config = readInput(vm.envOr("FOUNDRY_SCRIPT_CONFIG", string("")));
+        }
+    }
+    
+    /**
+     * @notice Use standard environment variables to load dependencies.
+     * @dev Will first check FOUNDRY_SCRIPT_DEPS_TEXT for raw json text.
+     *      Falls back to FOUNDRY_SCRIPT_DEPS for a standard file definition.
+     *      Finally will fall back to the given string `name`.
+     * @param name The default dependency file to load if no environment variables are set.
+     * @return dependencies The raw json text of the dependencies.
+     */
+    function loadDependencies(string memory name) internal returns (string memory dependencies) {
```

Comment:
Is your intention to keep readOutput?

Reply:
Yes I think it's helpful to have the low-level option. As I've said though this is temporary and will probably just be deprecated in favor of whatever `forge deploy` looks like.